plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/basic-setup.spec.js

```diff
@@ -28,9 +44,7 @@ test.describe(
 				.fill( '/product/' );
 			await page.locator( '#submit' ).click();
 			// Verify that settings have been saved
-			await expect(
-				page.locator( '#setting-error-settings_updated' )
-			).toContainText( 'Permalink structure updated.' );
+			await page.reload();
```

Comment:
Notification `#setting-error-settings_updated` is not showing on WPCOM and Pressable websites. Instead, my approach here is to refresh the page and check the `#permalink_structure` in the next step. 

WPCOM and Pressable:
<img width="829" alt="Screenshot 2024-08-25 at 15 36 04" src="https://github.com/user-attachments/assets/608e9d25-04c6-4f24-8100-8504bc4fd8ee">

wp-env:
<img width="679" alt="Screenshot 2024-08-25 at 15 35 58" src="https://github.com/user-attachments/assets/bbb29589-9ad2-4d3b-a8a2-1242c7948581">